chameleon/settings.py

```diff
@@ -1,4 +1,6 @@
 """
+Django settings file. This gets most values by loading a .env file.
```

Comment:
Don't know that we necessarily have to say the env variables are coming from a dotenv file, but as documentation this is fine!